crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -156,6 +156,18 @@ impl Default for StyledComponentsTransformConfigVc {
     }
 }
 
+/// Configuration options for the custom ecma transform to be applied.
+#[turbo_tasks::value(shared)]
+#[derive(Default, Clone)]
+pub struct CustomEcmaTransformPlugins {
```

Comment:
```suggestion
pub struct CustomEcmascriptTransformPlugins {
```

We use `Ecmascript` instead of `Ecma` everywhere